src/infra/api-key/csv/reader.go

```diff
@@ -18,15 +15,14 @@ const (
 	csvSeparator         = ','
 	csvCommentsMarker    = '#'
 	csvRowLen            = 4
-	csvHashOffset        = 0
+	csvAPIKeyOffset      = 0
```

Comment:
I dont get why this is called ApiKeyOffset when it is actually the ApiKeyHash that is located here

Reply:
done